accesscontroltool-package/src/main/META-INF/vault/filter.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-	<filter root="/apps/netcentric/actool"/>       
+	<filter root="/apps/netcentric/actool"/>  
+	<filter root="/home/users/system/netcentric/actool/acl/ynUbs6nnylcpRBs0eXdv"/>
+	<filter root="/" mode="merge"/>

```

Comment:
@santiagozky I don't think we can leave it like this - this will traverse the whole content tree and hence take way to long on production.